src/DEdge.Cardidy/Cardidy.cs

```diff
@@ -46,7 +46,8 @@ public static class Cardidy
         new UzCard(),
         new DinersClub(),
         new DinersClubUsAndCanada(),
-        new GPN()
+        new GPN(),
+        new BORICA()

```

Comment:
Can you move BORICA to be above GPN? The goal would be to have BORICA then GPN has output for `2205`. Since BORICA is most specific than GPN we want it to take the precedence. Thank you for your work :)